posix/ioctl.c

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/bits/device.h>
+#include <openenclave/corelibc/errno.h>
+#include <openenclave/corelibc/stdarg.h>
+#include <openenclave/internal/posix/device.h>
+#include <openenclave/internal/posix/fdtable.h>
+#include <openenclave/internal/posix/raise.h>
+#include <openenclave/internal/trace.h>
+
+int __oe_ioctl(int fd, unsigned long request, uint64_t arg)
+{
+    int ret = -1;
+    static const unsigned long _TIOCGWINSZ = 0x5413;
+
+    if (request == _TIOCGWINSZ)
+    {
+        static const unsigned long _TIOCGWINSZ = 0x5413;
+
+        if (request == _TIOCGWINSZ)
```

Comment:
I could be missing something, but isn't this if on line 21 the same as the if on line 17?

Reply:
Thank you for finding this Radhika. Fixed.